dcap_quoteverify/types/src/lib.rs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2022 The MobileCoin Foundation
+
+//! Rust wrappers for SGX DCAP quoteverify types
+
+#![no_std]
+
+#[cfg(test)]
+#[macro_use]
+extern crate yare;
+
+use mc_sgx_dcap_quoteverify_sys_types::sgx_qv_path_type_t;
+
+#[derive(Eq, PartialEq, Debug)]
+pub enum Path {
+    QVE,
+    QPL,
+}
+
+impl From<sgx_qv_path_type_t> for Path {
+    fn from(p: sgx_qv_path_type_t) -> Self {
+        match p {
+            sgx_qv_path_type_t::SGX_QV_QVE_PATH => Self::QVE,
+            sgx_qv_path_type_t::SGX_QV_QPL_PATH => Self::QPL,
+            p => panic!("Unexpected path: {:?}", p),
+        }
+    }
+}
+
+impl From<Path> for sgx_qv_path_type_t {
+    fn from(p: Path) -> sgx_qv_path_type_t {
+        match p {
+            Path::QVE => sgx_qv_path_type_t::SGX_QV_QVE_PATH,
+            Path::QPL => sgx_qv_path_type_t::SGX_QV_QPL_PATH,
+        }
+    }
+}
+
+#[cfg(test)]
+mod test {
+    extern crate std;
+    use super::*;
+
+    ide!();
+
+    #[parameterized(
```

Comment:
thoughts on using [yare](https://docs.rs/crate/yare/latest) for parameterized tests?
I know this enum is fairly simple, but for things like the return codes it may be nicer to utilize more values.

Reply:
Yeah, that lib is fine.